lib/ansible/plugins/action/reboot.py

```diff
@@ -178,6 +188,14 @@ def deprecated_args(self):
 
     def get_system_boot_time(self, distribution):
         boot_time_command = self._get_value_from_facts('BOOT_TIME_COMMANDS', distribution, 'DEFAULT_BOOT_TIME_COMMAND')
+        if self._task.args.get('boot_time_command'):
+            boot_time_command = self._task.args.get('boot_time_command')
```

Comment:
```suggestion
            boot_time_command = self._task.args.get('boot_time_command')
            try:
                check_type_str(boot_time_command, allow_conversion=False)
            except TypeError as e:
                raise AnsibleError("Invalid value given for 'boot_time_command': %s." % to_native(e))
```
i think we could put this to the ```if``` block to avoid extra checks of the default boot_time_command value

Reply:
Good suggestion. Thank you for the review.

Reply:
now i absolutely forgot what we discussed here but YW :)